src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -27,35 +28,45 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
             throw new InvalidArgumentError();
         }
 
-        return this._paypalScriptLoader.loadPaypalCommerce(paypalOptions.clientId)
-            .then(paypal => {
-                return paypal.Buttons({
-                    createOrder: () => this._setupPayment(),
-                    onApprove: data => this._tokenizePayment(paymentMethod.id, data),
-                }).render(`#${options.containerId}`);
+        return this._store.dispatch(this._checkoutActionCreator.loadDefaultCheckout())
+            .then(state => {
+                const config = state.config.getStoreConfig();
+                const cart = state.cart.getCart();
+
+                if (!config) {
+                    throw new MissingDataError(MissingDataErrorType.MissingCheckoutConfig);
+                }
+
+                const paramsScript = {
+                    'client-id': paypalOptions.clientId,
+                    currency: config.currency.code,
```

Comment:
Do you want to pass the store's default currency code or the currency code of the cart?

Reply:
@davidchin , we are planning to pass transactional currency (the one that later will be pushed to PayPal API)

Reply:
@davidchin , agree, I have double-checked on it - for the case when transactional currency differs from display one, in my case display CAD -> transactional USD, in cart.currency we have the correct one - transactional USD
![image](https://user-images.githubusercontent.com/51989411/76015235-9a926180-5f23-11ea-855f-f181437b93d3.png)
@helga1507  - can you please apply these changes

Reply:
@BC-ACherednichenko Sure